src/adaptors/adaptor_common.c

```diff
@@ -183,21 +228,33 @@ void qd_free_adaptor_config(qd_adaptor_config_t *config)
     free_qd_adaptor_config_t(config);
 }
 
-#define CHECK() if (qd_error_code()) goto error
+#define CHECK() \
+    if (qd_error_code()) goto error
 
-qd_error_t qd_load_adaptor_config(qd_dispatch_t *qd, qd_adaptor_config_t *config, qd_entity_t* entity, qd_log_source_t *log_source)
+qd_error_t qd_load_adaptor_config(qd_dispatch_t       *qd,
+                                  qd_adaptor_config_t *config,
+                                  qd_entity_t         *entity,
+                                  qd_log_source_t     *log_source)
 {
     qd_error_clear();
-    config->name    = qd_entity_opt_string(entity, "name", 0);                 CHECK();
-    config->host    = qd_entity_get_string(entity, "host");                    CHECK();
-    config->port    = qd_entity_get_string(entity, "port");                    CHECK();
-    config->address = qd_entity_get_string(entity, "address");                 CHECK();
-    config->site_id = qd_entity_opt_string(entity, "siteId", 0);               CHECK();
-    config->ssl_profile_name  = qd_entity_opt_string(entity, "sslProfile", 0); CHECK();
-    config->authenticate_peer = qd_entity_opt_bool(entity, "authenticatePeer", false); CHECK();
-    config->verify_host_name  = qd_entity_opt_bool(entity, "verifyHostname", false);   CHECK();
-
-    int hplen = strlen(config->host) + strlen(config->port) + 2;
+    config->name = qd_entity_opt_string(entity, "name", 0);
+    CHECK();
+    config->host = qd_entity_get_string(entity, "host");
+    CHECK();
+    config->port = qd_entity_get_string(entity, "port");
+    CHECK();
+    config->address = qd_entity_get_string(entity, "address");
+    CHECK();
+    config->site_id = qd_entity_opt_string(entity, "siteId", 0);
+    CHECK();
+    config->ssl_profile_name = qd_entity_opt_string(entity, "sslProfile", 0);
+    CHECK();
+    config->authenticate_peer = qd_entity_opt_bool(entity, "authenticatePeer", false);
+    CHECK();
+    config->verify_host_name = qd_entity_opt_bool(entity, "verifyHostname", false);
+    CHECK();
```

Comment:
probably disable formatter for this? or make the expression be parameter to check()?